content/docs/command-reference/plots/index.md

```diff
@@ -260,3 +277,52 @@ file:///Users/usr/src/plots/classes.csv.html
 
 > A confusion matrix [template](/doc/command-reference/plots#plot-templates) is
 > predefined in DVC (found in `.dvc/plots/confusion.json`).
+
+### Example for the HTML template
+
+There may be times when you need to produce plots without Internet access, or
+want to customize the plots output to put some extra content, like banners or
+extra text. DVC allows to replace the HTML file that contains the final plots.
+
+Download the Vega libraries into the directory where you'll produce the
+`dvc plots`.
+
+```dvc
+$ wget https://cdn.jsdelivr.net/npm/vega@5.10.0 -O my_vega.js
+$ wget https://cdn.jsdelivr.net/npm/vega-lite@4.8.1 -O my_vega_lite.js
+$ wget https://cdn.jsdelivr.net/npm/vega-embed@6.5.1 -O my_vega_embed.js
+```
+
+Create the following HTML file with your text editor:
```

Comment:
```suggestion
Create the following HTML file and save it in `.dvc/plots/mypage.html`:
```

Reply:
I'm not 100% sure about the location. As long as it doesn't cause confusion with regular plots templates... But if not I'd put it somewhere in the workspace and not in `.dvc/` I think.

Reply:
> if not I'd put it somewhere in the workspace and not in .dvc/ I think

Except that "the path supplied to `dvc config plots.html_template` is relative to `.dvc/` directory" 🤔 so yeah Idk. Up to you

Reply:
The references in `<script>` tags should also be in the _output_ directory. So if the user wants their plots output in `~/my-project`, the JS files should be there, and keeping this template in `~/my-project/` is a minor nuisance/clutter then. Preferably, these all should be in `~/my-project/plots/` or something like that but this becomes a bit irrelevant to us, so I decided to put everything under project root.

Reply:
> The references in <script> tags should also be in the output directory

OK so should we use just `./mypage.html` in this example? (updated suggestions)

> I decided to put everything under project root.

I'm confused. Project root is `.`. The example uses `.dvc/plots/`.

> See also https://github.com/iterative/dvc.org/pull/2472#discussion_r664776851

Reply:
Changed as your original suggestion. Thank you.